### DISCUSSION TITLE
Setting up GitHub action for automated tests

### DISCUSSION THREAD

**Discussion by AliMuhammadOfficial** [esa/opengeode#85] [Q&A] [answered]
Hi @maxime-esa,
I want to setup automated tests with github actions but I am facing some issues can guide me about that. 

I am facing bellow issue whenever I run the job!

import antlr3  # NOQA
    ModuleNotFoundError: No module named 'antlr3'

**Comment by maxime-esa** [comment, marked as answer]
You miss the dependency (`apt install python3-antlr3` in debian)
In principle all dependencies are installed when you run `make dependencies`
